Interfaces/About.html

```diff
@@ -0,0 +1,135 @@
+            <h1 class="openingText1">
+              Quarter mboko rappers           
+            </h1>
+            
+                 <h2 class="captureText1">
+                  About Mboko gods among us 2022
+                 </h2>
+            
+  
+                 <p class="explaining-text1">
+                    Mboko drill unearths Cameroon's hidden artists. It is a place where talent is discovered as others come to do what they do best in their careers. It began in Bamenda in 2015 and has since grown to become both national and international..
+                 </p>
+  
+                 <p class="eventDate1">Please contact us via email for any further question about mboko drill 2022</p>
+                 <span class="venue1">mboko@drill.com</span>
+          </div>
+        </section>
+        
+        <section class="about-logo-container" >
+
+            <div class="text-time">
+           <h2  class="text-header2">Mboko drill gods among us 2022</h2>
+
+           <hr class="line2">
+            </div>
+
+            <div class="about-text">
+                <p>The logo of mboko drill gods among us was decided through the logo competition of from the last in 2020</p>
+            </div>
+
+            <div class="about-logo">
+                <img src="../Images/logo_image/logo-image.png">
+            </div>
+
+        </section>
+
+        <section class="recent-section">
+            <div class="text-time">
+                <h2 class="text-header2">See the past mboko drill event</h2>
+     
+                <hr class="line2">
+                 </div>
+     
+                 <div class="about-text2">
+                     <p>Take a look at the recent past two events about mboko drill gods among us.</p>
+                 </div>
+
+                 <div class="recent-carts">
+                    <div class="card card1">
+                        <h1>2020</h1>
+                        <p>mboko drill in Douala</p>
+                    </div>
+                    <div class="card card2">
+                        <h1>2018</h1>
+                        <p>mboko drill in Yaounde</p>
+                    </div>
+                 </div>
+        </section>
+
+
+        <section class="spnosor-section">
+            <div class="text-time">
+                 <h3  class="text-header">Spnosors</h3>
+                 <hr class="line">
+            </div>
+            <div class="spnosor-container">
+              <img class="spnosor-imgages" src="../Images/Musician/MTN_Logo.svg.png" alt=" ">
+              <img class="spnosor-imgages" src="../Images/Musician/58429064a6515b1e0ad75aba.png" alt=" ">
+              <img class="spnosor-imgages" src="../Images/Musician/airtel-logo-vector-removebg-preview.png" alt=" ">
+              <img class="spnosor-imgages" src="../Images/Musician/camtel.jpg" alt=" ">
+              <img class="spnosor-imgages" src="../Images/Musician/Glo_button-removebg-preview.png" alt=" ">
```

Comment:
Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image